changelog/_unreleased/2020-09-17-change-the-customer-email-type.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Change the customer-email-type in admin
+issue: NEXT-9189
+author: Alexander Wink
+author_email: a.wink@kellerkinder.de
+author_github: @jinnoflife
+___
+# Administration
+* Adjust type of input field (email) for the customer email
```

Comment:
please use the allowed keywords for the changelog :wink: 

https://github.com/shopwareBoostDay/platform/tree/master/changelog

Reply:
done